### PR TITLE
Fix current benchmarks

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

Mitigates the issue we have seen in our current benchmarks #6747 that we overwhelm the exporters, which cause at the end that zeebe goes out of disk space on high load. The load is reduced to 200, which seems to work for now.

The tags have been set to SNAPSHOT instead of zeebe, because these tags are builds from 0.24.x which are not working anymore without current grpc protocol. SNAPSHOT seem to work, but idk when this was build. It is currently also not build by our pipeline. https://github.com/camunda-cloud/zeebe/issues/5608

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

**Comment by Zelldon** [comment]
With 200 it is still growing. 

proc
gen2
gen

**Review by npepinpe — APPROVED** [review]
bors r+

**Comment by npepinpe** [comment]
One other option would be to have the new benchmark script build and push images using the sha1 as tag.

EDIT: ah, just saw your comment. Guess I reviewed too quickly :smile:

**Comment by npepinpe** [comment]
Another option is to not have sustained throughput, but periods of sustained throughput. But that makes the starter more complicated =/

**Comment by Zelldon** [comment]
I try now with 150. If this doesnt work then Idk I dont want to go lower :sweat_smile:

**Comment by Zelldon** [comment]
> EDIT: or not...we can run the starter as a cron job. Run it for 1 hour, stop it for 15-20 min. Run it for another hour. etc.

This is an interesting idea tbh :D

File: benchmarks/setup/default/simpleStarter.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: simple
-        image: gcr.io/zeebe-io/starter:zeebe
+        image: gcr.io/zeebe-io/starter:SNAPSHOT
```
Comment (korthout) [inline benchmarks/setup/default/simpleStarter.yaml:19]: This will break the `.createBenchmark.sh` and setup medic benchmark scripts

**Comment by Zelldon** [comment]
I think as soon we have a restart our exporters can't keep up.

load

**Comment by Zelldon** [comment]
Ok I think I will go with 200 pi/s for now. It seems to be quite stable - the disk size seem to be under control. 
disk

If we take a look at the last 15 min the exporting seems to be okay'isch

processing

But if you check last 24 hours then it looks bad - I think the issue is related to adding metric counts on reprocessing (after restarts).

processing24


@korthout thanks for the pointer. I will fix that.

**Pull request merged**